flash_attn/cute/flash_fwd_sm100.py

```diff
@@ -1243,6 +1267,12 @@ def softmax_loop(
                 tStP_r2t=tStP_r2t,
                 sScale=sScale,
                 stage=stage,
+                batch_idx=batch_idx,
+                head_idx=head_idx,
+                m_block=m_block * 2 + stage,
```

Comment:
want to figure out statically if this should always be *2 not sure if its possible to have another work mapping that breaks this